index.html

```diff
@@ -35,21 +35,21 @@
           class="nav-search--input nav-search--box"
           placeholder="Search..."
         />
-        <button class="nav-search--button">
-          <i class="fas fa-search"></i>
-        </button>
+        <button class="nav-search--button">Search</button>
```

Comment:
It's recommended for `<button>` to have `type` attribute. If it will be controlled from client-side scripts,  `type="button"` is recommended ;)